### ISSUE TITLE
Refresh token not set to user object

### ISSUE THREAD

**Issue by kalel2** [Drenso/symfony-oidc#6]
Hello,
Seems there is a problem with set refresh token. I using this for a symfony project, and i found that refresh token is not set to user object, looked on source code and found the problem in src/OidcTokens.php in counstructor that it check a 'refresh_tokens' instead of 'refresh_token'
```
if (isset($tokens->refresh_tokens)) {
      $this->refreshToken = $tokens->refresh_tokens;
}
```
is it correct or maybe it's a bug?

**Comment by bobvandevijver** [comment]
Thanks for reporting this, you're absolutely right! This went wrong when I refactored the constructor in https://github.com/Drenso/symfony-oidc/commit/1f81b3a0f1ebcde51be5bf8f7b7a7c2d19691e24.

I will be releasing a fix in a couple of moments.

**Comment by bobvandevijver** [comment]
Fix released as v1.11.4

**Comment by kalel2** [comment]
Cool, thanks)
